net-proxy/tayga/metadata.xml

```diff
@@ -1,8 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<maintainer type="person">
-<email>heroxbd@gentoo.org</email>
-<name>Benda Xu</name>
-</maintainer>
+	<maintainer type="person">
+		<email>heroxbd@gentoo.org</email>
+		<name>Benda Xu</name>
+	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>zhuyifei1999@gmail.com</email>
+		<name>YiFei Zhu</name>
```

Comment:
I'd suggest moving yourself to the top of maintainer list, so it will give you more privilege such as bugs will be assigned to you by default

Reply:
Done.